index.bs

```diff
@@ -187,13 +191,15 @@ partial interface XRSession {
+Each {{XRSession}} has an <dfn for="XRSession">environment blending mode</dfn> value, which is an enum which MUST be set to whichever of the following values best matches the behavior of imagery presented by the session to the user.
 
-  - A blend mode of <dfn enum-value for="XREnvironmentBlendMode">opaque</dfn> indicates that the user's surrounding environment is not visible at all. Alpha values in the {{XRRenderState/baseLayer}} will be ignored, with the compositor treating all alpha values as 1.0. The {{XRSession/environmentBlendMode}} MUST NOT be {{opaque}} for {{XRSessionMode/"immersive-ar"}} sessions.
+  - If the pixels of the {{XRRenderState/baseLayer}} are composited using <a>source-over</a> blending with an opaque black color, the blend mode MUST indicate of <dfn enum-value for="XREnvironmentBlendMode">opaque</dfn>. {{XRSession/environmentBlendMode}} MUST NOT be {{opaque}} for {{XRSessionMode/"immersive-ar"}} sessions.
 
-  - A blend mode of <dfn enum-value for="XREnvironmentBlendMode">additive</dfn> indicates that the user's surrounding environment is visible and the {{XRRenderState/baseLayer}} will be shown additively against it on the device's primary displays. On those displays, alpha values in the {{XRRenderState/baseLayer}} will be ignored, with the compositor treating all alpha values as 1.0. This means that black pixels will appear fully transparent, and there is no way to make a pixel appear fully opaque. Secondary system usage of the app's views, such as a photo capture or video capture stream, MAY make use of the alpha channel.
+  - If the pixels of the {{XRRenderState/baseLayer}} are composited using <a>lighter</a> blending with the <a>user's environment</a>, the blend mode MUST indicate <dfn enum-value for="XREnvironmentBlendMode">additive</dfn>.
 
-  - A blend mode of <dfn enum-value for="XREnvironmentBlendMode">alpha-blend</dfn> indicates that the user's surrounding environment is visible and the {{XRRenderState/baseLayer}} will be blended with it according to the alpha values of each pixel. Pixels with an alpha value of 1.0 will be fully opaque and pixels with an alpha value of 0.0 will be fully transparent.  When an [=xr device=] [=perform automatic composition=] using a [=passthrough camera=] in an {{XRSessionMode/"immersive-ar"}} session, it MUST report {{XREnvironmentBlendMode/"alpha-blend"}} blending behavior.
+  - If the pixels of the {{XRRenderState/baseLayer}} are composited using <a>source-over</a> blending with the <a>user's environment</a>, the blend mode MUST indicate <dfn enum-value for="XREnvironmentBlendMode">alpha-blend</dfn>.
+
+The <dfn>user's environment</dfn> is an opaque buffer that contains a pixel representation of the current world. If the device renders in stereo, this buffer SHOULD be the world representation with reference to the <a>eye</a>.
```

Comment:
This paragraph doesn't seem quite right given how the definition is used on line 198.  Perhaps I'm misreading it, but I'm getting the impression that by explicitly calling out a "buffer" there's an implication of camera pixels with doesn't jive with an "additive" blend mode.

Reply:
maybe remove the reference to buffer and pixel:

> ...is an opaque representation of the current world